Doc/library/decimal.rst

```diff
@@ -350,18 +350,19 @@ Decimal objects
    *value* can be an integer, string, tuple, :class:`float`, or another :class:`Decimal`
    object. If no *value* is given, returns ``Decimal('0')``.  If *value* is a
    string, it should conform to the decimal numeric string syntax after leading
-   and trailing whitespace characters, as well as underscores throughout, are removed::
-
-      sign           ::=  '+' | '-'
-      digit          ::=  '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'
-      indicator      ::=  'e' | 'E'
-      digits         ::=  digit [digit]...
-      decimal-part   ::=  digits '.' [digits] | ['.'] digits
-      exponent-part  ::=  indicator [sign] digits
-      infinity       ::=  'Infinity' | 'Inf'
-      nan            ::=  'NaN' [digits] | 'sNaN' [digits]
-      numeric-value  ::=  decimal-part [exponent-part] | infinity
-      numeric-string ::=  [sign] numeric-value | [sign] nan
+   and trailing whitespace characters are removed:
+
+   .. productionlist:: decimal
+      sign: "+" | "-"
+      digit: "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
+      indicator: "e" | "E"
+      digits: (`digit` | "_")* `digit` (`digit` | "_")*
+      decimalpart: `digits` "." [`digits`] | ["."] `digits`
+      exponentpart: indicator [sign] digits
+      infinity: "Infinity" | "Inf"
+      nan: "NaN" [`digits`] | "sNaN" [`digits`]
+      numericvalue: `decimalpart` [`exponentpart`] | `infinity`
```

Comment:
Do Sphinx production lists support `numeric-value` instead or should it be without any weird symbol? if not, use `_` instead of `-` to separate words.

Reply:
Sphinx does not support `-`, but it does support `_`.

Reply:
Yet another thing I should perhaps add to my list of things to fix upstream... Thanks for checking.